backend/index.ts

```diff
@@ -27,7 +27,7 @@ metricsServer.listen(METRICS_PORT, () => {
 
 (async () => {
   try {
-    // Wait 15 seconds to give the Grafana container extra time to spin up and be ready to respond
+    // Wait 10 seconds to give the Grafana container extra time to spin up and be ready to respond
```

Comment:
Oops! I forgot to change this comment when I changed the timer to 10 seconds. Great catch.